cmd/deploy.go

```diff
@@ -214,7 +219,7 @@ func buildPushDockerImage(c config.Context, name, path, nextTag, cloudDomain str
 	}
 
 	image, tag := docker.GetImageTagFromParsedFile(cmds)
-	if config.CFG.ShowWarnings.GetBool() && !validImageRepo(image) {
+	if config.CFG.ShowWarnings.GetBool() && !validAirflowImageRepo(image) && !validRuntimeImageRepo(image) {
```

Comment:
If I understand correctly, if the user's Dockerfile's FROM statement does not use the official astro runtime image (quay.io/astronomer/astro-runtime), we still allow the user to deploy but just show a warning and ask for confirmation, right ?

So the goal is to allow customers to use custom images that extend the astro-runtime image right ?

Reply:
yup, that's correct